recipes/sentry-native/all/conanfile.py

```diff
@@ -120,8 +117,6 @@ def requirements(self):
                 self.requires("crashpad/cci.20220219")
             else:
                 self.requires("zlib/[>=1.2.11 <2]")
-                if self.settings.os in ("Linux", "FreeBSD"):
-                    self.requires("libcurl/[>=7.78.0 <9]")
```

Comment:
What's the motivation of no longer building crashpad_util library?

Reply:
I don't understand your question.

Reply:
@uilianries I don't understand how this PR is related to crashpad_util library not being built?

Reply:
Sorry, I guess commented in the wrong PR.